tools/testing/e2e/CMakeLists.txt

```diff
@@ -0,0 +1,47 @@
+iree_add_all_subdirs()
+
+
+iree_cc_binary(
+  NAME
+    iree-e2e-matmul-test
+  SRCS
+    "iree-e2e-matmul-test.cc"
+    "test_utils.cc"
+  DEPS
+    iree::base
+    iree::base::internal
+    iree::base::internal::cpu
+    iree::base::internal::flags
+    iree::base::internal::path
+    iree::hal
+    iree::modules::hal
+    iree::tooling::context_util
+    iree::tooling::device_util
+    iree::vm
+    iree::vm::cc
+)
+
+iree_cc_binary(
+  NAME
+    iree-e2e-conv2d-test
+  SRCS
+    "iree-e2e-conv2d-test.cc"
+    "test_utils.cc"
+  DEPS
+    iree::base
+    iree::base::internal
+    iree::base::internal::cpu
+    iree::base::internal::flags
+    iree::base::internal::path
+    iree::hal
+    iree::modules::hal
+    iree::tooling::context_util
+    iree::tooling::device_util
+    iree::vm
+    iree::vm::cc
+)
+
+add_executable("iree-e2e-matmul-test" IMPORTED GLOBAL)
+add_executable("iree-e2e-conv2d-test" IMPORTED GLOBAL)
+set_property(TARGET "iree-e2e-matmul-test" PROPERTY IMPORTED_LOCATION "${CMAKE_CURRENT_BINARY_DIR}/iree-e2e-matmul-test")
```

Comment:
@ScottTodd, what's the right way to `add_executable`? The iree_e2e_generated_test didn't recognize this TARGET, so I added these lines. Is there any better way?

Reply:
You may be bumping into https://github.com/openxla/iree/blob/623c1c88a581dfd9fff17314352c8000f2071e43/tools/CMakeLists.txt#L53-L58  Could use the fully qualified target name instead of this unqualified name.

You can debug target names with `_DEBUG_IREE_PACKAGE_NAME` here: https://github.com/openxla/iree/blob/623c1c88a581dfd9fff17314352c8000f2071e43/build_tools/cmake/iree_macros.cmake#L327-L329

Reply:
Thanks. It was really helpful for debugging.